timestep.c

```diff
@@ -19,6 +19,18 @@
  */
 
 
+/* variables for organizing PM steps of discrete timeline */
+static struct time_vars {
```

Comment:
It is the duration of the span. The typedef is a bit superfluous since it is only used once.